content/docs/tools/teddyCloud/supported-server-hardware.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Supported NAS/Server hardware"
+description: "list of supported NAS/server hardware (incomplete)"
+bookCollapseSection: true
+---
+# Supported NAS/Server hardware
+
+Teddycloud is supported on nearly all servers utilizing the x86_64, arm64, and armhf (linux/arm/v7) architectures, as Docker containers are available for these platforms.
+Below is a compilation of NAS/server hardware that has been successfully utilized to host TeddyCloud:
+
+| NAS / Server hardware           | Limitations                                                                                                                                                                 |
+|---------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Raspberry Pi 4                  | -                                                                                                                                                                           |
+| Raspberry Pi Zero 2             | slow on initial startup (expect 20 minutes and more for initial certificate creation!), encoding also very slow, possible crashes in some situations, but in general usable |
+| Synology NAS DS923+ (DSM 7.2.1) | -                                                                                                                                                                           |
+| ...                             |                                                                                                                                                                             |
+
+If you're utilizing alternative NAS/server hardware configurations, your contribution is welcome. Feel free to share your experience and insights with the community.
+
+## Tested Hardware, but not supporting TeddyCloud
+
+
+| NAS / Server hardware | Comment/Discussion/Links                                                                            |
+|-----------------------|-----------------------------------------------------------------------------------------------------|
+| Raspberry Pi Zero     | https://forum.revvox.de/t/hardware-requirements-does-teddycloud-run-on-a-raspberry-zero-1st-gen/309 |
```

Comment:
The problem here is docker not TeddyCloud itself. This should be clarified (it may be possible to directly run the binary out of the docker image)

Reply:
Yes, that's why I linked the thread and encourage people to move the entry to working setup if they have one after the table.

Reply:
So at least if someone looks on it, he can go directly to the discussion.

Reply:
But I will make it more clearer that this seems to be a docker problem